apps/test.py

```diff
@@ -116,19 +116,25 @@ def _get_user_application(self, username, app_name):
         apps_qs = Application.objects.filter(name__exact=app_name).filter(user__username=username)
         return apps_qs.first()
 
-    def create_token(self, first_name, last_name):
+    def create_token(self, first_name, last_name, bene_id=None, hicn_hash=None, mbi_hash=None):
         passwd = '123456'
         user = self._create_user(first_name,
                                  passwd,
                                  first_name=first_name,
                                  last_name=last_name,
+                                 fhir_id=bene_id if bene_id is not None else settings.DEFAULT_SAMPLE_FHIR_ID,
+                                 user_hicn_hash=hicn_hash if hicn_hash is not None else self.test_hicn_hash,
+                                 user_mbi_hash=mbi_hash if mbi_hash is not None else self.test_mbi_hash,
                                  email="%s@%s.net" % (first_name, last_name))
-        if Crosswalk.objects.filter(_fhir_id=settings.DEFAULT_SAMPLE_FHIR_ID).exists():
-            Crosswalk.objects.filter(_fhir_id=settings.DEFAULT_SAMPLE_FHIR_ID).delete()
+        fhir_id = bene_id if bene_id is not None else settings.DEFAULT_SAMPLE_FHIR_ID
+
+        if Crosswalk.objects.filter(_fhir_id=fhir_id).exists():
+            Crosswalk.objects.filter(_fhir_id=fhir_id).delete()
+
         Crosswalk.objects.create(user=user,
-                                 fhir_id=settings.DEFAULT_SAMPLE_FHIR_ID,
-                                 user_hicn_hash=self.test_hicn_hash,
-                                 user_mbi_hash=self.test_mbi_hash)
+                                 fhir_id=fhir_id,
+                                 user_hicn_hash=hicn_hash if hicn_hash is not None else self.test_hicn_hash,
+                                 user_mbi_hash=mbi_hash if mbi_hash is not None else self.test_mbi_hash)
```

Comment:
I suggest refactoring this section:
- Suggest sticking with the parameter `fhir_id` rather than `bene_id` since that is how its referred to in upstream functions (may need to be changed elsewhere in integration test calls).
- Suggest setting defaults for `fhir_id`, `user_hicn_hash`, and `user_mbi_hash` once at the top rather than duplicating efforts in each function call.

Reply:
- renamed parameter name to fhir_id per comment (also renamed a local var (which happen to use name fhir_id) to avoid shadowing the parameter.
- the 'new' sample A (fhir_id=-88888888888888) which includes each of the EOB profiles DME, HHA, SNF, Hospice, Carrier, PDE, Inpatient, Outpatient is introduced to unblock downstream components e.g. BB2 testing EOB profiles, the 'old' sample A (fhir_id=-20140000008325) does not have DME, HHA, SNF, Hospice.
- many existing tests are still using old sample A with user first name "John", last name "Doe" to create crosswalk entry for tests, we are not going to touch that part and the corresponding pre-calculated test_mbi_hash/test_hicn_hash will remain the same 
- the new test test_eob_profiles_endpoint which will use fhir_id=-88888888888888, mbi_hash=SAMPLE_A_888_MBI_HASH & hicn_hash=SAMPLE_A_888_HICN_HASH,  and user first name = sample_a_88888888888888 and last name = sample_a_88888888888888 to create a separate crosswalk to 'new' sample A EOB profiles test.
- so crosswalk & token creation code is reused with additional func parameters exposed, where legacy tests remain un-changed (still using old sample A), new EOB profiles test override with 'new' sample A fhir_id, user name, mbi/hicn hash
- my guess is that eventually the 'new' sample A will be merged into sample A which is fhir_id=-20140000008325, since that beneficiary is already part of the Developer IG.